pkg/ebpf/events_pipeline.go

```diff
@@ -170,8 +169,16 @@ func (t *Tracee) decodeEvents(outerCtx context.Context, sourceChan chan []byte)
 					t.handleError(fmt.Errorf("failed to read argument %d of event %s: %v", i, eventDefinition.Name, err))
 					continue
 				}
-
-				args = append(args, trace.Argument{ArgMeta: argMeta, Value: argVal})
+				if args[idx].Value != nil {
+					t.handleError(fmt.Errorf("read more than one instance of argument %s of event %s", arg.Name, eventDefinition.Name))
```

Comment:
It might be helpful to also include the previous and current argument values in the message